core/src/main/resources/error/error-classes.json

```diff
@@ -668,6 +668,24 @@
       }
     }
   },
+  "LOCATION_ALREADY_EXISTS" : {
+    "message" : [
+      "Cannot create the location <location> because it already exists.",
+      "Choose a different path or remove the existing location."
```

Comment:
Given the subclass messages, why do we need this sentence?
We are basically giving advise first, and then adding more info afterwards... with more advise seems weird

Reply:
Location and table are different levels of abstractions. And this advise are about resolving the issue on different levels. The main message in the main error class tells users about the location in a filesystem - remove the location if it was created in a filesystem using some fs tools behind of Spark, for example. The sub-class messages focus on more specific problem, and tell users how to solve the issue on the table name level.

> with more advise seems weird

If we can advise something that can potentially solve user's issue even the chance is low, we should do that, I believe.

Probably, the advise in this particular case is trivial but I added it because I thought the main error class **can be used alone** w/ linking to a managed table.

Reply:
I didn't mean to remove advise. I means to collect all the "facts" up front and then collect all the "advise" in the end.
I.e. in the most trivial solution just move "Choose a different path or remove the existing location." int the suberror message.... Anyway, these are all nits...

Reply:
> I.e. in the most trivial solution just move

ok. I'll move it.